web/src/components/product/ProductSelectionPage.tsx

```diff
@@ -102,6 +107,10 @@ function ProductSelectionPage() {
   const registration = useRegistration();
   const { products, selectedProduct } = useProduct({ suspense: true });
   const [nextProduct, setNextProduct] = useState(selectedProduct);
+  // FIXME: should not be accepted by default first selectedProduct is accepted
+  // because it's a singleProduct iso.
```

Comment:
this can be really issue with s390 SLES iso as it is single product. So I would ask in advance